domain.yml

```diff
@@ -537,6 +547,8 @@ templates:
 actions:
   - utter_ask_faq_tutorials
   - utter_greet
+  - utter_greet_name
```

Comment:
we'll leave this here, but for educational purposes: you don't actually need to add all templates if they're only called from an action and not used in the stories :D

Reply:
good to know! You just mean in the list actions, right? That makes sense.

Reply:
exactly